File: eBeertijaBackend/eBeertijaBackend/DTOs/NewRacunDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ebeertijaBackend.DTOs
{
    public class NewRacunDto
    {
        public int Broj { get; set; }
        public string BrojOpis { get; set; }
        public List<StavkaCjenikaDto> StavkeCjenikaNaSnazi { get; set; }
        public List<StavkaRacunaDto> StavkeRacuna { get; set; }
        public int StolId { get; set; }
        public string OznakaStola { get; set; }
    }
}

File: eBeertijaFrontend/src/app/Models/NewRacunDto.ts
import { StavkaCjenikaDto } from './StavkaCjenikaDto';
import { StavkaRacunaDto } from './StavkaRacunaDto';

export class NewRacunDto {
    broj: number;
    brojOpis: string;
    stavkeCjenikaNaSnazi: StavkaCjenikaDto[];
    stavkeRacuna: StavkaRacunaDto[];
    stolId: number;
    oznakaStola: string;
    ukupnoSaPdvom?: number;
}
